src/common/pf_ppm.h

```diff
@@ -92,6 +92,9 @@ int pf_ppm_set_iocs (
 
 /**
  * Retrieve the data and IOPS for a sub-module.
+ *
+ * This is for reading back the data and the IOPS you have set to the PPM.
```

Comment:
A bit too personal and a repetition of previous string.

Reply:
Changed to 
`Note that this is not from the PLC, but previously set by the application.`
Done